### DISCUSSION TITLE
how to acesss arguments in ursina entity in Circle Mode

### DISCUSSION THREAD

**Discussion by Marcilio-prog** [pokepetter/ursina#430] [Q&A]
Hello! I´m trying to create an simulation of double slit diffraction.I created the entity
e1=Entity(model=Circle(resolution=64,mode='line',radius=0.23,thickness=3),color=color.white,position=(x_dir,y_top-5.25,z0))
 and need to change the radius.How can I get this? The dot operation e1.circle or e1.model.radius does not work
Thanks!

**Comment by pokepetter** [comment]
Can't you just scale the Entity?
Setting radius on the model should work though, but you'd have to call .generate() to update the model.

**Comment by Marcilio-prog** [comment]
It is not a natural solution, but it is an idea.Thanks.
